_pages/what's-your-i-am-statement.html

```diff
@@ -5,111 +5,55 @@
   </div>
-  <div class="row push-top push-bottom iam-journey">
-    <div class="container">
-      <div class="col-md-12 text-center">
-        <h2>Questions/Comments</h2>
-      </div>
-      <div class="col-md-6">
-        <h4>What’s a Journey</h4>
-        <p>The Journey is a six-week self-discovery process. Thousands of people (kids and adults) come together to
-          focus on a topic to kick-start spiritual growth.</p>
-        <p>It’s comprised of three components:</p>
-        <ul>
-          <li>the weekend message</li>
-          <li>individual challenges in the app</li>
-          <li>group discussion</li>
-        </ul>
-        <p>All of these work together to make the journey one of the most transformational things we do.</p>
-      </div>
-      <div class="col-md-6">
-        <h4>What materials do I need?</h4>
-        <p>Download the <strong>Crossroads Anywhere App</strong> from the Apple or Google Play stores to get your
-          individual and group content. If you already have the app, make sure it is up to date. The guides will be
-          available the first week of the Journey.</p>
-        <p>If you don't have an iOS or Android device, download the <a href="https://s3.amazonaws.com/crds-cms-uploads/IAmJourneyGuide.pdf"
-            target="_blank">I Am Journey Guide here</a>. For additional materials essential to the Journey <a href="/materials/">click
-            here</a>.</p>
-        <p>Have problems with the app? <a href="mailto:appsupport@crossroads.net">Email us</a></p>
-      </div>
-      <div class="col-md-12 text-center push-top">
-        <p>Check out the full <a href="/faq/">FAQ.</a><br>Or email us: <a href="mailto:journey@crossroads.net">Journey@crossroads.net</a></p>
-      </div>
-    </div>
+  <div class="col-sm-8 col-sm-offset-2 text-center">
+    <p class="lead push-top">Looking for the journey materials? <a href="https://www.crossroads.net/iamjourney2">Visit
+        the journey page</a></p>
   </div>
 </div>
-<script src="https://crossroads-media.s3.amazonaws.com/documents/i-am-wordcloud/jquery.jqgoogleforms.min.js" type="text/javascript"></script>
 <script type="text/javascript">// <![CDATA[
-  $(document).ready(function () {
-    $(".iam-form-submit").click(function (e) {
-      e.preventDefault();
-      var googleForm = $(this).jqGoogleForms({ "formKey": "1FAIpQLSd48e9FRQjPMH3SjXCc7NbnwJICF11FODGa5POuA_HgMSmwww" });
+  document.addEventListener('deferred-js-ready', function () {
+
+ var head = document.head;
+    var script = document.createElement('script');
+    script.type = 'text/javascript';
+    script.src = 'https://crossroads-media.s3.amazonaws.com/documents/i-am-wordcloud/jquery.jqgoogleforms.min.js';
+    head.appendChild(script);
+
+
+    $(document).ready(function () {
+      $(".iam-form-submit").click(function (e) {
+        e.preventDefault();
+        var googleForm = $(this).jqGoogleForms({ "formKey": "1FAIpQLSd48e9FRQjPMH3SjXCc7NbnwJICF11FODGa5POuA_HgMSmwww" });
```

Comment:
assuming this form key is cool to checkin?

Reply:
Deleting the 2 "I am pages" as they are "no longer of value".  One of them was the one with the key.